cmd/main.go

```diff
@@ -3,13 +3,22 @@ package main
 import (
 	"NKSS-backend/config"
 	"NKSS-backend/pkg"
+	"NKSS-backend/pkg/auth"
 	"fmt"
+	"os"
 )
 
 func main() {
 	config := config.GetConfig()
 	app := pkg.NewApp(&config)
 
+	arglen := len(os.Args[1:])
+	
+	if arglen > 0 && os.Args[1] == "generate-jwt" {
+		fmt.Println("Generated jwt: ", auth.CreateJWT(os.Args[2], os.Args[3], config.HMACSecret))
+		return
+	}
+
```

Comment:
can this be in a separate program? say `cmd/genjwt.go`

Reply:
Okay, have done so. I didn't think it was appropriate to make too many changes to other files not in cmd, so I just took env variables directly instead of creating an abstraction in ```config.go```